src/Drawables/Drawable.vala

```diff
@@ -262,17 +262,17 @@ public class Akira.Drawables.Drawable {
         // Draw the circle and preserve the visual properties.
         context.stroke ();
 
-        var lenght = 10 / scale;
+        var length = 10 / scale;
         // Create the horizontal stroke.
         context.new_path ();
         context.move_to (-lenght, 0);
-        context.line_to (lenght, 0);
+        context.line_to (length, 0);
         context.stroke ();
 
         // Create the vertical stroke.
         context.new_path ();
         context.move_to (0, -lenght);
-        context.line_to (0, lenght);
+        context.line_to (0, length);
```

Comment:
please review closely

Reply:
Thanks for the typos.
You need to replace all instances of `lenght` with `length`, even those in `context.move_to (-lenght, 0);`, otherwise the build is busted.

Reply:
Done. Not sure how I missed those.